Compilers.sln

```diff
@@ -182,6 +182,7 @@ Global
 		src\Analyzers\VisualBasic\Analyzers\VisualBasicAnalyzers.projitems*{2531a8c4-97dd-47bc-a79c-b7846051e137}*SharedItemsImports = 5
 		src\Workspaces\SharedUtilitiesAndExtensions\Compiler\VisualBasic\VisualBasicCompilerExtensions.projitems*{2531a8c4-97dd-47bc-a79c-b7846051e137}*SharedItemsImports = 5
 		src\Analyzers\Core\Analyzers\Analyzers.projitems*{275812ee-dedb-4232-9439-91c9757d2ae4}*SharedItemsImports = 5
+		src\Dependencies\Collections\Microsoft.CodeAnalysis.Collections.projitems*{275812ee-dedb-4232-9439-91c9757d2ae4}*SharedItemsImports = 5
```

Comment:
>src\Dependencies\Collections\Microsoft.CodeAnalysis.Collections.projitems*{275812ee-dedb-4232-9439-91c9757d2ae4}*SharedItemsImports = 5 [](start = 2, length = 135)

Is this an intentional change?

Reply:
IDE has tended to add this implicitly when using certain refactorings in recent weeks. IIRC we saw this change come in another PR and decided to just merge it. Merging latest main into the feature branch will probably cause this diff to go away.

Reply:
Thanks. Reverted

---
In reply to: [588362066](https://github.com/dotnet/roslyn/pull/51494#discussion_r588362066) [](ancestors = 588362066)